src/sql-parser/src/parser.rs

```diff
@@ -2927,7 +2927,8 @@ impl Parser {
 
     fn parse_tail(&mut self) -> Result<Statement, ParserError> {
         let name = self.parse_object_name()?;
-        let with_snapshot = if self.parse_keyword("WITH") {
+
+        let without_snapshot = if self.parse_keyword("WITHOUT") {
```

Comment:
I think Albert's latest opinion on slack was to also allow "WITH SNAPSHOT", and have that be an alias for the default behavior

Reply:
apparently thats "the SQL way"? it also has the nice property of being backwards compatible